### ISSUE TITLE
How should I change my training scheme if I want to use HRNet V2 to train pose estimation mission?

### ISSUE THREAD

**Issue by ArchNew** [leoxiaobin/deep-high-resolution-net.pytorch#90]
I've noticed, even though there are not many differences between V1 and V2 other than the way they mix the last layers (or lack of), the batch norm's momentum is 0.01 in V2 versus 0.1 in V1. Should I use 0.01 momentum if I want to use V2 to train pose mission? Are there other parameters I should change in order to get similar results as V1 in the pose mission?
